src/skmultiflow/trees/arf_hoeffding_tree.py

```diff
@@ -1,12 +1,12 @@
 from skmultiflow.utils import check_random_state
-from skmultiflow.trees.hoeffding_tree import HoeffdingTree, MAJORITY_CLASS, \
+from skmultiflow.trees.hoeffding_tree import HoeffdingTreeClassifier, MAJORITY_CLASS, \
     NAIVE_BAYES
 from skmultiflow.trees.nodes import RandomLearningNodeClassification
 from skmultiflow.trees.nodes import RandomLearningNodeNB
 from skmultiflow.trees.nodes import RandomLearningNodeNBAdaptive
 
 
-class ARFHoeffdingTree(HoeffdingTree):
+class ARFHoeffdingTree(HoeffdingTreeClassifier):
```

Comment:
Given that `ARFHoeffdingTree` is the base estimator of `AdaptiveRandomForestClassifier` (and `AdaptiveRandomForestRegressor` is coming on the way), should not its name carry the "classifier" sufix?

Reply:
Agree, renamed in a445c3716bba3a579434808008bad0b215109e6b